webapp/scripts/preview/requestBody.js

```diff
@@ -193,7 +193,12 @@ HeadersTab.prototype = domplate(TabView.Tab.prototype,
         var url = this.file.request.url;
         if (url) {
             var anchor = $(body).find("a")[0];
-            anchor.setAttribute("href", url);
+            if (url.includes("<REDACTED>")) {
+              anchor.setAttribute("href", "");
+              anchor.setAttribute("class", anchor.getAttribute("class") + " disableLink");
```

Comment:
I was curious if there was a vanilla JS equivalent to `addClass` and found this which might be a bit simpler: https://stackoverflow.com/questions/507138/how-to-add-a-class-to-a-given-element
```suggestion
              anchor.classList.add("disableLink");
```